apps/examples/testcase/le_tc/network/tc_net_pbuf.c

```diff
@@ -39,7 +39,7 @@
 * @brief		:check for the memory allocation for the pbuf by transport layer
 * @scenario		:
 * @apicovered		:pbuf_alloc()
-* @precondition 	:
+* @precondition		:
```

Comment:
1. As I left before, please split commit for formatting. You should not include both renaming and formatting.
2. Because of space and tab mixing, it shows coding rule violation. And because of using tab, it shows different alignment at each system. That's why I prefer using spaces instead of tabs.

Reply:
yes. i'll split the commit.